src/app/context/loading.context.tsx

```diff
@@ -0,0 +1,21 @@
+'use client';
```

Comment:
It’d be better to put this loading state into the `news.context` because it’s news’ loading state and not some global one